### PR TITLE
Platform display embed API

### PR DESCRIPTION
**Pull request by andycreeth**

### CODE REVIEW

File: app/services/platform-apps/api/modules/display.ts [hunk 1]
```diff
@@ -0,0 +1,118 @@
+import { Module, apiMethod, EApiPermissions, IApiContext } from './module';
+import { Display } from 'services/video';
+import uuid from 'uuid/v4';
+import electron from 'electron';
+import { Subscription } from 'rxjs/Subscription';
+
+interface IDisplayCreateOptions {
+  position: IVec2;
```
Comment (holiber) [inline app/services/platform-apps/api/modules/display.ts]: when I see `position` I always have a question, is it position of the left-top corner of the center of the screen?

File: app/services/platform-apps/api/modules/display.ts [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  size: IVec2;
+  sourceId?: string;
```
Comment (holiber) [inline app/services/platform-apps/api/modules/display.ts]: Why `sourceId` is optional?

File: app/services/platform-apps/api/modules/display.ts (hunk 1 above)
Comment (andycreeth) [reply app/services/platform-apps/api/modules/display.ts]: Everything in SLOBS uses top-left as the origin currently.

File: app/services/platform-apps/api/modules/display.ts (hunk 2 above)
Comment (andycreeth) [reply app/services/platform-apps/api/modules/display.ts]: If you omit `sourceId` the display will be the entire video output.

**Pull request merged**